modules/ossm-load-test-results.adoc

```diff
@@ -7,11 +7,11 @@ This module included in the following assemblies:
 = Load test results
 
 The upstream Istio community load tests mesh consists of *1000* services and *2000* sidecars with 70,000 mesh-wide requests per second.
-Running the tests using Istio 1.6.8, generated the following results:
+Running the tests using Istio 1.12.3, generated the following results:
```

Comment:
Istio version 1.14.1?

Reply:
@pbajjuri20 Make sure to look at the upstream 1.12 doc page: https://istio.io/v1.12/docs/ops/deployment/performance-and-scalability/

Reply:
@jwendell : thanks for the clarification